src/System/include/BipedalLocomotion/System/Advanceable.h

```diff
@@ -36,6 +36,13 @@ template <class _Input, class _Output> class BipedalLocomotion::System::Advancea
     using Input = _Input;
     using Output = _Output;
 
+    Advanceable() = default;
```

Comment:
This shouldn't be required

Reply:
Removed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/397/commits/1fcb4d038bd83d56b79ed3116034fce9109f03f6

Reply:
Looks like it's needed afterall!